src/modules/core-milestone/__test__/core-milestone.test.tsx

```diff
@@ -0,0 +1,140 @@
+// npx jest ./src/modules/core-milestone/__tests__/core-milestone.test.tsx
```

Comment:
The name of the folder should be `__tests__` instead of `__test__`

Notice the plural form of the word test

Reply:
i have change folder name